algoliasearch/build.gradle

```diff
@@ -33,10 +33,11 @@ ext {
     - Log-in on sonatype.org, go to staging profile, select com.algolia, id = last part of URL */
 }
 
-
-nexusStaging {
-    packageGroup = PUBLISH_GROUP_ID
-    stagingProfileId = PUBLISH_STAGING_ID
+if (project.hasProperty("nexus")) {
```

Comment:
Great. This plugin does not do much when just applying and not executed. I don't know if there is much gain when surrounded with an `if` block (and there is no warning as it is applied on the root project).